src/content/blog/dev-portal.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "The Astro Developer Portal"
+description: "Today, we’re launching the Astro Developer Portal. A platform, where theme authors and creators, can fully create, own and update their themes built for Astro."
```

Comment:
Small suggestion. It could also maybe hint towards the theme catalogue?
```suggestion
description: "Today, we’re launching the Astro Developer Portal, a platform where theme authors and creators can create, own and update their themes built for Astro."
```